config/karma.conf.js

```diff
@@ -162,6 +162,9 @@ export function getKarmaConfig(karma : Karma, cfg : Config) : KarmaConfig {
                 {
                     type: 'text'
                 },
+                {
```

Comment:
Could we change the existing html open to lcov? According to the docs, lcov includes both html and lcov: https://karma-runner.github.io/0.8/config/coverage.html. That way we don't have to use lcovonly
```
{
                    type:   'lcov',
                    dir:    'coverage/',
                    subdir: '.'
                }
```